tests/build/build.go

```diff
@@ -123,6 +126,190 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 		})
 
+		When("a new simple build component is created", Label("annotations"), func() {
```

Comment:
Is the PR still in progress? I don't see the changes covering other scenarios covering advanced build supporting new annotations.